outplacement/models/outplacement.py

```diff
@@ -56,6 +56,18 @@ def _read_group_employee_ids(self, employees, domain, order):
     meeting_remote = fields.Selection(selection=[('no','On Premice'),('yes','Remote')],string='Meeting type')
     uniq_ref = fields.Char(string='Uniq Id', size=64, trim=True, )
 
+    status = fields.Selection(string="status", selection=[
```

Comment:
Use outplacement.state for this

Reply:
I can't find outplacement.state, did you mean outplacement.stage_id?

Reply:
Yes